flag_generated.go

```diff
@@ -6,17 +8,15 @@ import (
 	"time"
 )
 
-// WARNING: This file is generated!
-
 // BoolFlag is a flag with type bool
 type BoolFlag struct {
-	Name        string
-	Usage       string
-	EnvVar      string
-	FilePath    string
-	Required    bool
-	Hidden      bool
-	Destination *bool
+	Name		string
+	Usage		string
+	EnvVar		string
+	FilePath	string
+	Required	bool
+	Hidden		bool
+	Destination	*bool
```

Comment:
Nice 💯 was this spacing done automatically?

Reply:
Oh yes :)

Reply:
👏👏👏